components/com_finder/src/Model/SearchModel.php

```diff
@@ -42,6 +45,23 @@ class SearchModel extends ListModel
 	 */
 	protected $searchquery;
 
+	/**
+	 * Maps each sorting field with a text label.
+	 *
+	 * @var string[]
+	 * @since __DEPLOY_VERSION__
+	 */
+	protected $sortOrderFieldsLabels = [
+		'relevance.asc' => 'COM_FINDER_SORT_BY_RELEVANCE_ASC',
+		'relevance.desc' => 'COM_FINDER_SORT_BY_RELEVANCE_DESC',
+		'title.asc' => 'JGLOBAL_TITLE_ASC',
+		'title.desc' => 'JGLOBAL_TITLE_DESC',
+		'date.asc' => 'JDATE_ASC',
+		'date.desc' => 'JDATE_DESC',
+		'price.asc' => 'COM_FINDER_SORT_BY_PRICE_ASC',
+		'price.desc' => 'COM_FINDER_SORT_BY_PRICE_DESC'
```

Comment:
Multiline array should have a trailing comma on last item in the array

```suggestion
		'price.desc' => 'COM_FINDER_SORT_BY_PRICE_DESC',
```